.circleci/main-config.yml

```diff
@@ -125,6 +125,9 @@ commands:
             ?(*_)develop)
               DEPLOY_ENV=dev
             ;;
+            DDP-?(*))
```

Comment:
Could you please explain what does it mean?

Reply:
It is matching the branch name (e.g., DDP-1235_Whatever) This is saying that if the branch name has the DDP- prefix, we will target the dev environment.